tdvt/tdvt/config_gen/templates.py

```diff
@@ -197,6 +197,14 @@
                         'bool_underscore' : True
                  },
 
+    #[tactile-pulsar-824].[TestV1].[Calcs]
```

Comment:
Try expressing this section in the `googlebigquery.ini` file instead of adding it to the global list.

```
[Datasource]
Name = googlebigquery
LogicalQueryFormat = TEST_googleBigQuery

...
[LogicalConfig]
Name = TEST_googleBigQuery

tablename = $dsName
fieldnameNoSpace = True
tablePrefix = [tactile-pulsar-824].[TestV1].

```

Reply:
Fixed :))